### ISSUE TITLE
Pop-up Window

### ISSUE THREAD

**Issue by ashleydentino** [mamont77/vixi#60]
I'd love to implement a pop-up window on my site with an opt-in box (for first name and email) that adds them to my mailchimp list. 

Here are a few examples we can model:
http://www.marieforleo.com/
http://brendonburchard.com/
http://thechalkboardmag.com/

I will send you the text, images and design we can use. Please let me know the first step. Thanks!

**Comment by mamont77** [comment]
Do you mean this pop-up?
![odesk team room 04-18-2015 07 39 00](https://cloud.githubusercontent.com/assets/970967/7214031/375576ce-e59e-11e4-8eb3-36b4795d3d59.png)
It's first step ))
>>I will send you the text, images and design we can use.

**Comment by ashleydentino** [comment]
Can we add content inside of it, similar to the examples I sent over to you?

**Comment by mamont77** [comment]
Yes, we can. I think i will have time tommorow. Sorry, i am writing from
phone.
20 апр. 2015 г. 19:00 пользователь "Ashley Dentino" <
notifications@github.com> написал:

**Comment by ashleydentino** [comment]
Ok, I will design the content. Do I choose the size? Is there anything
specific you need from me to get started?

On Mon, Apr 20, 2015 at 12:08 PM, Ruslan <notifications@github.com> wrote:

> Yes, we can. I think i will have time tommorow. Sorry, i am writing from
> phone.
> 20 апр. 2015 г. 19:00 пользователь "Ashley Dentino" <
> notifications@github.com> написал:
>
> > Can we add content inside of it, similar to the examples I sent over to
> > you?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/mamont77/vixi/issues/60#issuecomment-94493354>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mamont77/vixi/issues/60#issuecomment-94495028>.
>



-- 
Ashley Dentino | www.vixi.com
email. ashley.dentino@gmail.com | cell. 602.315.9229

facebook <http://www.facebook.com/ashley.dentino> | instagram
<http://instagram.com/ashleydentino> | free newsletter
<http://vixi.us2.list-manage.com/subscribe?u=9298c5763dd416b1743c949fa&id=6e47921471>

........
Celebrate your thoughts; they become words. Celebrate your words; they
become actions.
 Celebrate your actions; they become habits. Celebrate your habits; they
become character.
Celebrate your character; for it becomes your destiny.

**Comment by mamont77** [comment]
Can you provide me the text and background image only? I am going to make
in css.
20 апр. 2015 г. 19:12 пользователь "Ashley Dentino" <
notifications@github.com> написал:

> Ok, I will design the content. Do I choose the size? Is there anything
> specific you need from me to get started?
>

**Comment by mamont77** [comment]
Do I choose the size? 

Width about 840px. It is desirable but not necessary.
The picture should not detract from the text input by the user.

**Comment by mamont77** [comment]
Hello Ashley,

I received an email with the file. Thank you.
Frankly I'm confused now.
For what the website to implement it? :-)
It seems this new functionality should duplicate http://vixi.com/ and "get the goodness" form.
![vixi feel alive - google chrome 04-23-2015 07 45 49](https://cloud.githubusercontent.com/assets/970967/7290726/e011c078-e98c-11e4-9f01-a284415224cd.png)
However, I would suggest that you wrote about another site http://viximeals.com/ and and functionality similar to "Register" form.
![odesk team room 04-18-2015 07 39 00](https://cloud.githubusercontent.com/assets/970967/7290748/1d5a4036-e98d-11e4-82aa-f838a992406a.png)
Just specify where to do it.
Also write how long the delay in seconds before the window will be shown to the user. If there is a page exceptions (front, not front, blog) - also specify, if need.

**Comment by ashleydentino** [comment]
Sorry to confuse you. The pop-up will be on vixi.com. The goal is to have a
better conversion rate on my email list with the traffic that comes to my
site.

Can we have it display to only first-time visitors of the site? We can set
the delay time at 10 seconds.

Yes, it will use the same first name and email address field you
highlighted in red. I would like to do a new rectangular "Subscribe Now"
button instead of the round "get the goodness" button. We can use the same
style as the button on the homepage for the blog post teasers. Let's make
the color a yellow (instead of black).

Is there a way to only show the pop-up for first time visitors?

Thank you!

On Thu, Apr 23, 2015 at 12:51 AM, Ruslan <notifications@github.com> wrote:

> Hello Ashley,
>
> I received an email with the file. Thank you.
> Frankly I'm confused now.
> For what the website to implement it? :-)
> It seems this new functionality should duplicate http://vixi.com/ and
> "get the goodness" form.
> [image: vixi feel alive - google chrome 04-23-2015 07 45 49]
> <https://cloud.githubusercontent.com/assets/970967/7290726/e011c078-e98c-11e4-9f01-a284415224cd.png>
> However, I would suggest that you wrote about another site
> http://viximeals.com/ and and functionality similar to "Register" form.
> [image: odesk team room 04-18-2015 07 39 00]
> <https://cloud.githubusercontent.com/assets/970967/7290748/1d5a4036-e98d-11e4-82aa-f838a992406a.png>
> Just specify where to do it.
> Also write how long the delay in seconds before the window will be shown
> to the user. If there is a page exceptions (front, not front, blog) - also
> specify, if need.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mamont77/vixi/issues/60#issuecomment-95431601>.
>



-- 
Ashley Dentino | www.vixi.com
email. ashley.dentino@gmail.com | cell. 602.315.9229

facebook <http://www.facebook.com/ashley.dentino> | instagram
<http://instagram.com/ashleydentino> | free newsletter
<http://vixi.us2.list-manage.com/subscribe?u=9298c5763dd416b1743c949fa&id=6e47921471>

........
Celebrate your thoughts; they become words. Celebrate your words; they
become actions.
 Celebrate your actions; they become habits. Celebrate your habits; they
become character.
Celebrate your character; for it becomes your destiny.

**Comment by mamont77** [comment]
Sorry for the delay, was the weekend.

Ready for testing.

The result:

![vixi feel alive - google chrome 04-27-2015 18 18 34](https://cloud.githubusercontent.com/assets/970967/7350788/f5255abc-ed09-11e4-9e02-a9d22d0811f6.png)

**Comment by ashleydentino** [comment]
Perfect. Thank you.

On Mon, Apr 27, 2015 at 11:19 AM, Ruslan <notifications@github.com> wrote:

> Sorry for the delay, was the weekend.
>
> Ready for testing.
>
> The result:
>
> [image: vixi feel alive - google chrome 04-27-2015 18 18 34]
> <https://cloud.githubusercontent.com/assets/970967/7350788/f5255abc-ed09-11e4-9e02-a9d22d0811f6.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mamont77/vixi/issues/60#issuecomment-96701077>.
>



-- 
Ashley Dentino | www.vixi.com
email. ashley.dentino@gmail.com | cell. 602.315.9229

facebook <http://www.facebook.com/ashley.dentino> | instagram
<http://instagram.com/ashleydentino> | free newsletter
<http://vixi.us2.list-manage.com/subscribe?u=9298c5763dd416b1743c949fa&id=6e47921471>

........
Celebrate your thoughts; they become words. Celebrate your words; they
become actions.
 Celebrate your actions; they become habits. Celebrate your habits; they
become character.
Celebrate your character; for it becomes your destiny.

**Comment by ashleydentino** [comment]
Hey Ruslan - Are the blog post teaser images on my homepage showing up for you? They are not for me. Also, can we make the fields on the pop up window for name and email a little big longer and the text inside of them a little big lighter (like on the homepage opt-in). Thanks!

**Comment by mamont77** [comment]
>>Are the blog post teaser images on my homepage showing up for you? 

Fixed.
It is https://github.com/mamont77/vixi/issues/58#issuecomment-100192880 not working right now. I will find the best way in the next few days.

>>can we make the fields on the pop up window for name and email a little big longer 

Done. The result:
![vixi feel alive - google chrome 05-08-2015 13 49 18](https://cloud.githubusercontent.com/assets/970967/7535021/3b900e2a-f589-11e4-88aa-3db477e68c0f.png)
